c/parallel/include/cccl/c/for.h

```diff
@@ -39,14 +39,10 @@ CCCL_C_API CUresult cccl_device_for_build(
   const char* cub_path,
   const char* thrust_path,
   const char* libcudacxx_path,
-  const char* ctk_path) noexcept;
+  const char* ctk_path);
 
 CCCL_C_API CUresult cccl_device_for(
-  cccl_device_for_build_result_t build,
-  cccl_iterator_t d_data,
-  int64_t num_items,
-  cccl_op_t op,
-  CUstream stream) noexcept;
+  cccl_device_for_build_result_t build, cccl_iterator_t d_data, int64_t num_items, cccl_op_t op, CUstream stream);
```

Comment:
Seems like an inconsistent style change?

Reply:
This is VSCode dictated change, not sure why. Removal of `noexcept` result in this reformatting.
I can manually reformat it to have each function argument on a separate line.

Reply:
No worries then!